### PR TITLE
enable Utility feature

### PR DESCRIPTION
**Pull request by rahulpaul217**
request to merge this utility to main branch

### CODE REVIEW

File: config.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+config 1
```
Comment (rahulpaulgmailuser) [inline config.txt:1]: This looks bag review

File: util/utility.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+utility 1
```
Comment (rahulpaulgmailuser) [inline util/utility.txt:1]: Do not eaxisy

**Review by rahulpaulgmailuser — CHANGES_REQUESTED** [review]
Please fix

File: util/utility.txt (hunk 2 above)
Comment (rahulpaul217) [reply util/utility.txt:1]: ok

Comment (rahulpaul217) [reply util/utility.txt:1]: done

**Pull request merged**